src/bin/julialauncher.rs

```diff
@@ -393,9 +395,16 @@ fn run_app() -> Result<i32> {
             }
 
             // replace the current process
-            std::process::Command::new(julia_path)
-                .args(&new_args)
-                .exec();
+            if is_valid_julia_path(&julia_path) {
```

Comment:
Hm, this will essentially add a complete Julia launch into the path before we _actually_ start Julia, right? I think that would add too much overhead.

I would assume that we can instead just handle the error if the `exec` command doesn't succeed?

Reply:
Your suggestion seems to work!